### ISSUE TITLE
[Floating] Bug with current user rank

### ISSUE THREAD

**Issue by nikitamorozov** [playgameservices/play-games-plugin-for-unity#2407]
Method
`SocialFunctions.GetScores(SocialFunctions.GetLeaderBoardName(GameModeString), (score, scores, users) => {
	var i = 0;
	foreach (var scored in scores)
	{
		Debug.log("It's me on " + i + " - " + (scored.rank == score.rank));
	}
});`

Sometimes condition (scored.rank == score.rank) works not for my account.

Bug happen when I bit my highscore, but works well after restart.

What it would be? Cache?

**Comment by olehkuznetsov** [comment]
Yes, there is internal cache. Will check if we can improve this.

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
